### DISCUSSION TITLE
How to install on Elementary os 6 base 20.04

### DISCUSSION THREAD

**Discussion by andirsun** [Prince781/vala-language-server#206] [Q&A] [answered]
**Describe the bug**

I cant find the packages for base 20.04 in elementary os 6

The main repo has some errors
```
W: No Hash entry in Release file /var/lib/apt/lists/partial/download.opensuse.org_repositories_home:_Prince781_xUbuntu%5f20.04_InRelease
E: Failed to fetch http://download.opensuse.org/repositories/home:/Prince781/xUbuntu_20.04/Packages  404  Not Found [IP: 195.135.221.134 80]
E: Some index files failed to download. They have been ignored, or old ones used instead.

```

**Comment by Prince781** [comment, marked as answer]
I'm disabling the Ubuntu 20.04 repo (and Ubuntu 20.10) for now since the version of Vala needed by VLS isn't currently shipped. However, I spoke to @danrabbit about distributing VLS in eOS 6's repository and he agreed to it. So that should be available soon.

For now you'll just have build VLS from source. Let me know if you need any help doing that.

**Reply by Prince781** [reply]
@andirsun UPDATE: Since a little over an hour ago VLS is now available in eOS 6's main repository. Just run `sudo apt update` and `sudo apt install vala-language-server`. Let me know if this doesn't work.

**Reply by andirsun** [reply]
Yes it works !
